### ISSUE TITLE
[A11Y]: accordion: `aria-expanded` should be on header not content

### ISSUE THREAD

**Issue by aghassemi** [ampproject/amphtml#5488]
currently `aria-expanded` is set on the content, it should be set on the `header` instead.

**Comment by aghassemi** [comment]
/to @dvoytenko you get an easy one since you have other _Duties_ to take care of :)
